contracts/argent_account.cairo

```diff
@@ -51,4 +52,25 @@ mod ArgentAccount {
     fn get_contract_address_test() -> felt {
         dummy_syscalls::get_contract_address() 
     }
+
+    fn assert_only_self_test() -> felt {
+        let a = 1;
+        asserts::assert_only_self();
+        a
+    }
+
+    fn assert_guardian_set_test() -> felt {
+        let a = 1;
+        let guardian = guardian::read();
+        asserts::assert_guardian_set(guardian);
+        a
+    }
+
+    fn assert_correct_tx_version_test(tx_version: felt) -> felt {
+        let a = 1;
+        asserts::assert_correct_tx_version(tx_version);
+        1
+    }
```

Comment:
pls let's not include testing code in the main contract, let's move it in the PR's description if not used immediately. same for @gaetbout's `fn get_contract_address_test`

Reply:
Was just to showcase how to call it.
But yep it should def be removed asap, I should have put that in the chat, my mistake

Reply:
Digging through chats can be cumbersome, better in the PR's description

Reply:
yeah sure will remove it